SimMuon/GEMDigitizer/src/GEMSimpleModel.cc

```diff
@@ -216,7 +218,10 @@ void GEMSimpleModel::simulateNoise(const GEMEtaPartition* roll, CLHEP::HepRandom
                                       + GE11ElecBkgParam1 * rollRadius
                                       + GE11ElecBkgParam2 * rollRadius * rollRadius
                                       + GE11ElecBkgParam3 * rollRadius * rollRadius * rollRadius;
+      
+    // Scale up/down for desired instantaneous lumi (reference is 5E34, double from config is in units of 1E34)
     averageNoiseRatePerRoll = averageNeutralNoiseRatePerRoll + averageNoiseElectronRatePerRoll;
+    averageNoiseRatePerRoll *= instLumi_*rateFact_*1.0/5;
```

Comment:
weel that is not a magic number: the functions we implemented in the code for describing the neutron background are already normalized to the 5x10^34 Hz/cm^2. So that 5 is needed if we want to rescale giving as input the new instantaneous luminosity. I am not sure if it is a good thing to make it a parameter because the functions are something fixed... whoever updates the functions has to update also that number to make the whings work properly.